sourced/ml/transformers/basic.py

```diff
@@ -160,3 +172,17 @@ def deserialize_uast(self, row):
         row_dict = row.asDict()
         row_dict["uast"] = self.parse_uast(row.uast[0])
         yield Row(**row_dict)
+
+
+class ContentDeserializer(Transformer):
```

Comment:
You don't need this at all. The only reason we have UastDeserializer is that UASTs cannot be pickled - it is happening while persisting the pipeline, so we have to cache bytes and deserialize them every time.

In your case it is already bytes and they are perfectly pickleable.